File: ueueo-framework/ueueo-blob-storing/src/main/java/Abp/BlobStoring/BlobProviderArgs.cs
using System.Threading;
using JetBrains.Annotations;

namespace Volo.Abp.BlobStoring;

public abstract class BlobProviderArgs
{
    [NotNull]
    public String ContainerName;//  { get; }

    [NotNull]
    public BlobContainerConfiguration Configuration;//  { get; }

    [NotNull]
    public String BlobName;//  { get; }

    public CancellationToken CancellationToken;//  { get; }

    protected BlobProviderArgs(
        @Nonnull String containerName,
        @Nonnull BlobContainerConfiguration configuration,
        @Nonnull String blobName,
        CancellationToken cancellationToken = default)
    {
        ContainerName = Check.NotNullOrWhiteSpace(containerName, nameof(containerName));
        Configuration = Objects.requireNonNull(configuration, nameof(configuration));
        BlobName = Check.NotNullOrWhiteSpace(blobName, nameof(blobName));
        CancellationToken = cancellationToken;
    }
}

File: abp-frameworks/ueueo-blob-storing/src/main/java/com/ueueo/blobstoring/BlobProviderArgs.java
package com.ueueo.blobstoring;

import com.ueueo.threading.CancellationToken;
import org.springframework.lang.NonNull;

import java.util.Objects;

public abstract class BlobProviderArgs {
    @NonNull
    private String containerName;

    @NonNull
    private BlobContainerConfiguration configuration;

    @NonNull
    private String blobName;

    public CancellationToken cancellationToken;

    protected BlobProviderArgs(
            @NonNull String containerName,
            @NonNull BlobContainerConfiguration configuration,
            @NonNull String blobName,
            CancellationToken cancellationToken) {
        this.containerName = Objects.requireNonNull(containerName);
        this.configuration = Objects.requireNonNull(configuration);
        this.blobName = Objects.requireNonNull(blobName);
        this.cancellationToken = cancellationToken;
    }

    @NonNull
    public String getContainerName() {
        return containerName;
    }

    @NonNull
    public BlobContainerConfiguration getConfiguration() {
        return configuration;
    }

    @NonNull
    public String getBlobName() {
        return blobName;
    }

    public CancellationToken getCancellationToken() {
        return cancellationToken;
    }
}
